ipatests/test_webui/ui_driver.py

```diff
@@ -59,7 +59,9 @@
     NO_YAML = False
 except ImportError:
     NO_YAML = True
+
 from ipaplatform.paths import paths
+from ipaplatform.tasks import tasks
```

Comment:
ftrivino@localhost ~/src/freeipa/ipatests/test_webui (master)>grep paths * -R
ui_driver.py:from ipaplatform.paths import paths
ui_driver.py:            options.binary_location = paths.CHROMIUM_BROWSER

Reply:
ohh, you didn't remove paths, then it is ok.